ffs-shared/rule/src/commonMain/kotlin/rule/RuleEval.kt

```diff
@@ -507,5 +579,17 @@ private sealed class RuleExpr<out T> {
         private fun Iterator<*>.fastForward() {
             while (hasNext()) next()
         }
+
+        @Suppress("PARAMETER_NAME_CHANGED_ON_OVERRIDE")
+        private fun delegateRangeToCollection(
```

Comment:
```suggestion
        private fun wrapRangeInCollection(
```

Delegate has a “special” meaning in Kotlin, so I'd prefer we use an alternative term.